### DISCUSSION TITLE
Creating multiple workspaces

### DISCUSSION THREAD

**Discussion by computersrmyfriends** [langgenius/dify#4840] [General]
How do I create an additional workspace inside dify?
I have a workspace and I want to create another workspace

Thanks

**Comment by crazywoola** [comment]
Please contact our BD for this feature. business@dify.ai

**Comment by computersrmyfriends** [comment]
Why? Is it a paid feature?

**Reply by patryk20120** [reply]
Yes, you need to have enterprise license.

**Comment by aldrinjenson** [comment]
Hey, isn't this multi workspace feature available in the self-hosted version?

**Reply by patryk20120** [reply]
It is not.

**Reply by wsbao** [reply]
If we want to deploy a multi-workspace version within our organization, is it feasible to host it by ourselves? Of course, we can pay the enterprise license for it.

**Reply by jaypif** [reply]
It make sense for Dify Cloud to have multi-workspace under Licence.
However for the self hosted version, it must be possible to create several workspace.

Can you think to add this feature soon ? thanks

**Reply by hxt365** [reply]
The team will never merge the feature since they want to keep it a paid feature. Fork the code and add the feature yourself

**Reply by jimmy66688** [reply]
> The team will never merge the feature since they want to keep it a paid feature. Fork the code and add the feature yourself

Who's repo can I find the code?

**Comment by patricia-cmc** [comment]
Hi, I'm using Dify form AWS Marketplace. 
I have two questions.
1) I want to create additional Workspaces but I can't find the option. Isn't it available in the AWS marketplace image?
2) The version in AWS marketplace (0.6.13) is very different from the Community version (latest 0.15.3 ), Is it because it has a different features?

**Comment by matteo-rama** [comment]
it's been a while since the last comment, do the dev team come to the conclusion that this  feature isn't a "killer" feature and that even on prem we deserve that? :)
